mopidy_mpd/translator.py

```diff
@@ -106,22 +106,29 @@ def track_to_mpd_format(track, position=None, stream_title=None):
     if track.album and track.album.uri:
         result.append(("X-AlbumUri", track.album.uri))
 
-    result = [element for element in result if _has_value(*element)]
+    if tagtypes is None:
+        tagtypes = tagtype_list.TAGTYPE_LIST
```

Comment:
Either make this a set or make `tagtype_list.TAGTYPE_LIST` itself a set.

Reply:
Changed tagtype_list.TAGTYPE_LIST to a set.